app/models/task.py

```diff
@@ -1,5 +1,37 @@
-from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+from sqlalchemy import Integer, String, Text, DateTime, ForeignKey
 from ..db import db
 
 class Task(db.Model):
-    id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    __tablename__ = "tasks"
+
+    id:           Mapped[int]       = mapped_column(Integer, primary_key=True, autoincrement=True)
+    title:        Mapped[str]       = mapped_column(String, nullable=False)
+    description:  Mapped[str]       = mapped_column(Text, nullable=False)
```

Comment:
In the current version of SQLAlchemy `nullable=False` is the **default**, so we should leave off the `mapped_column` statement so we don't add unnecessary code to the codebase. 

```suggestion
    title: Mapped[str] 
    description: Mapped[str] 
```